haiku-apps/copynametoclipboard/copynametoclipboard-1.0.1.recipe

```diff
@@ -9,7 +9,8 @@ COPYRIGHT="2000 Werner Freytag
 	2012 Humdinger"
 LICENSE="MIT"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+SOURCE_URI="https://github.com/HaikuArchives/CopyNameToClipboard/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+SOURCE_DIR="CopyNameToClipboard-1da1d52429db53f15c3debf817e982d9ce6346b4"
```

Comment:
Good catch!
I would, however, only drop the parentheses, since it is OK to use $HOMEPAGE in SOURCE_URI.
While at it, you can change:
```
SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
```
to
```
srcGitRev="1da1d52429db53f15c3debf817e982d9ce6346b4"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
SOURCE_DIR="CopyNameToClipboard-$srcGitRev"
```
BTW, you forgot to add the CHECKSUM_SHA256.